### DISCUSSION TITLE
Pre-built binaries without UI

### DISCUSSION THREAD

**Discussion by neovintage** [ConduitIO/conduit#247] [Ideas]
Was thinking about development teams that want to deploy Conduit as a background service within their infrastructure. Is it worth offering pre-built binaries without the UI?

**Comment by hariso** [comment]
@neovintage I might be missing something but they can obviously run the binary with the UI, and simply not use the UI (the HTTP and gRPC APIs are exposed). Or maybe there's some value in leaving out the UI, such as a smaller binary size?

**Reply by neovintage** [reply]
The thought was having a smaller binary size. At this moment, it doesn't seem worthwhile to have that when you can start the current binaries without the UI.

**Reply by hariso** [reply]
I think you have a good point with UI-less builds. If we go this route, we shouldn't forget to update the Docker images too.

**Comment by raulb** [comment]
👋 I'm happy to announce this has been added as part of the latest release, here's a blog post announcing it https://meroxa.com/blog/introducing-the-new-conduit-cli:-a-powerful-tool-for-managing-your-pipelines/ and some related change-logs.

- https://conduit.io/changelog/2025-02-04-conduit-0-13-0-release
- https://conduit.io/changelog/2025-02-04-pipelines-exit-on-error/
- https://conduit.io/changelog/2025-02-06-conduit-0-13-1-release

No more UI in conduit. Long live the CLI.
